testing/kite/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Testing with Kite
+permalink: /testing/kite/
+---
+
+
+{% include toc-hide.html %}
+
+
+## This is Karoshi Interoperability Testing Engine (KITE)
+
+
+The effortless way to test WebRTC compliance, prevent [Karoshi](https://en.wikipedia.org/wiki/Kar%C5%8Dshi) with KITE!
+
+KITE is designed to be a generic, reusable and easy to maintain automated testing environment.
+
+So far, our developments on KITE allow interoperability testing on stable versions of all major web browsers, namely Chrome, Firefox, Safari and Edge, running on desktop OS, that is either Ubuntu, MacOS or Windows 10.
+
+Mobile testing (Chrome,Firefox on Android, Safari on iOS) is soon to be operational.
+
+### Development:
+
+Check out KITE on [webrtc/KITE public repository](https://github.com/webrtc/KITE).
+
+### Included sample IceConnectionTest test
+
+IceConnectionTest algorithm is as follows:
```

Comment:
Suggest adding before this:
"The ICE Connection test is a basic test showing that two browsers can establish a connection."